Assets/Scripts/DataModels/Missions/Mission.cs

```diff
@@ -53,7 +53,10 @@ public void ProcessOutcomes()
     {
         foreach (MissionOutcome outcome in outcomes)
         {
-            outcome.Process(this);
+            if (outcome != null)
+            {
+                outcome.Process(this);

```

Comment:
If we overload the Process method, then when we loop through the outcomes, how do we know whether to call the one with the Mission param vs. the other one? Some outcomes need to get the Mission and others don't.

Checking types and casting them or something seems ridiculous.

Could we just have the void method? Well maybe, but we need a way of referencing the Mission scriptable object from the individual outcomes. Seems kinda hard as theres no inheritance there, and looking them up in the database or something seems ridiculous too.